ui/apps/platform/src/services/ComplianceResultsService.ts

```diff
@@ -47,12 +47,13 @@ export type ListComplianceCheckClusterResponse = {
 export function getComplianceProfileCheckResult(
     profileName: string,
     checkName: string,
+    sortOption: ApiSortOption,
     page: number,
     perPage: number
 ): Promise<ListComplianceCheckClusterResponse> {
     const queryParameters = {
         query: {
-            pagination: { ...getPaginationParams(page, perPage) },
+            pagination: { ...getPaginationParams(page, perPage), sortOption },
```

Comment:
Later, you can update compliance service files as a chore after improvement in #11293